src/axom/inlet/Table.cpp

```diff
@@ -193,5 +217,97 @@ bool Table::required()
   return (bool)intValue;
 }
 
+ std::shared_ptr<Table> Table::registerVerifier(std::function<bool()> lambda) {
+  SLIC_WARNING_IF(m_verifier, fmt::format("Verifier for Table {0} already set", 
```

Comment:
Ok -- so it is a warning to register a second verifier, but the new one will then overwrite the old one.